tasks/jenkins.js

```diff
@@ -29,12 +29,6 @@ module.exports = function (grunt) {
     'jenkins:env',
     'rejectRejFiles',
 
-    // We need to build all the Kibana packages so ESLint is able to verify that
-    // e.g. imports resolve properly. If we don't build the packages here, the
-    // `main` field in their `package.json` would link to a location that
-    // doesn't exist yet.
-    'buildPackages',
```

Comment:
This is no longer needed because of the change to `kbn bootstrap` (it runs `kbn:bootstrap` scripts defined in `package.json` in all packages)